cmd/smith/app/app.go

```diff
@@ -123,6 +144,29 @@ func (a *App) Run(ctx context.Context) error {
 	if err != nil {
 		return err
 	}
+
+	// Events
+	eventBroadcaster := record.NewBroadcaster()
+	eventBroadcaster.StartLogging(logInfo)
+	eventBroadcaster.StartRecordingToSink(&core_v1client.EventSinkImpl{Interface: clientset.CoreV1().Events("")})
+	recorder := eventBroadcaster.NewRecorder(scheme, core_v1.EventSource{Component: "smith-controller"})
+
+	// Leader election
+	if a.LeaderElect {
```

Comment:
under what situations would you not want this. Does it cause a perf penalty? Also, why not default?

Reply:
The doc says it is experimental. We can make it an always-on thing once we are confident it works as expected.